fisco-bcos-tars-service/RpcService/RpcInitializer.cpp

```diff
@@ -109,8 +109,16 @@ bcos::rpc::RpcFactory::Ptr RpcInitializer::initRpcFactory(bcos::tool::NodeConfig
     auto gateway = std::make_shared<GatewayServiceClient>(
         gatewayPrx, gatewayServiceName, protocolInitializer->keyFactory());
 
-    auto factory = std::make_shared<bcos::rpc::RpcFactory>(_nodeConfig->chainId(), gateway,
-        protocolInitializer->keyFactory(), protocolInitializer->dataEncryption());
+    bcos::security::KeyEncryptInterface::Ptr keyEncryptionPtr = nullptr;
```

Comment:
this logic is same as `fisco-bcos-tars-service/GatewayService/GatewayInitializer.cpp`, please refactor to a function, maybe should be part of protocolInitializer

Reply:
fix it